src/test/functions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Functions extends TestCase
+{
+    public function testGenerateArtistHtmlSuccess()
+    {
+        $artistPlaceholder = ['name' => '', 'yearsLived' => '', 'image' => '', 'favouriteMedium' => '', 'knownFor' => '', 'placeOfBirth' => ''];
+        
+        $expectedOutput = '<h2>' . $artistPlaceholder['name'] . '</h2>' . 
```

Comment:
Did you copy paste the return statement of generateArtistHtml?  If yes, maybe your test would be more robust if your $expectedOutput was a plain string e.g. `$expectedOutput = <h2>Salvador Dali</h2>`. And it might be easier if the $artistsPlaceholder array had actual values in it and not empty strings?

Reply:
done😌